### PR TITLE
Add ability to list all transactions

### PR DESCRIPTION
**Pull request by jknipp**
Add missing functionality to retrieve transactions with additional parameters.

### CODE REVIEW

**Comment by jknipp** [comment]
@duff This is a pretty straightforward change to add the ability to list transactions, shouldn't be any surprises.

File: test/remote/list_transactions_test.exs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+    :timer.sleep(1000)
+    {:ok, _trans_1} = Spreedly.verify(env(), gateway.token, card.token)
+    :timer.sleep(1000)
+    {:ok, _trans_2} = Spreedly.purchase(env(), gateway.token, card.token, 100)
+
+    :timer.sleep(1000)
+    {:ok, list} = Spreedly.list_transactions(env(), order: :desc, count: 100)
+
+    # Concurrent tests make it difficult to ensure order,
```
Comment (duff) [inline test/remote/list_transactions_test.exs]: Looks like this test and the previous test are the same (for the concurrent reasons you mentioned). Makes me think that perhaps this test could be removed?

**Comment by duff** [comment]
👍

File: test/remote/list_transactions_test.exs (hunk 1 above)
Comment (jknipp) [reply test/remote/list_transactions_test.exs]: 👍 Makes sense.

**Pull request merged**